modules/templates/helper.go

```diff
@@ -186,6 +186,9 @@ func NewFuncMap() []template.FuncMap {
 		"ParseDeadline": func(deadline string) []string {
 			return strings.Split(deadline, "|")
 		},
+		"DefaultTheme": func() string {
+			return setting.UI.DefaultTheme
```

Comment:
I think that it may be better ti add this as variable to data since this function just acts like a variable.

Reply:
I'm not sure about this because this pattern already exists and I am just following. Example see `AppUrl` above in this file.